### ISSUE TITLE
Seth doesn't encode single uint when it's too big

### ISSUE THREAD

**Issue by krzkaczor** [dapphub/dapptools#67]
Seth doesn't encode single, big uints
```
seth calldata 'mint(uint256)' "10000000000"
```

Returns:
```
Cannot parse uint256

Caused by:
  number too large to fit in target type
```
`10000000000` obviously should fit into uint256 type.

On the other hand, encoding smaller uint works:
```
seth calldata 'mint(uint256)' "1000000"
0xa0712d6800000000000000000000000000000000000000000000000000000000000f4240
```

**Comment by gbalabasquer** [comment]
try `seth calldata 'mint(uint256)'  $(seth --to-uint256 10000000000)`

**Comment by krzkaczor** [comment]
Thanks, @gbalabasquer it works! it's weird that even though it's value is part of method signature I need to encode it separately... shouldn't it be done automatically?

**Comment by nanexcool** [comment]
Problem is with `ethabi` that won't handle large numbers. We need to look at alternatives...

**Comment by asymmetric** [comment]
See #107.

**Comment by MrChico** [comment]
Solved by #374
